packages/web/src/components/toolbarButton.css

```diff
@@ -16,15 +16,25 @@
 
 .toolbar-button {
   flex: none;
-  border: none;
-  outline: none;
   color: var(--vscode-sideBarTitle-foreground);
   background: transparent;
-  padding: 4px;
   margin-left: 10px;
-  cursor: pointer;
   display: inline-flex;
   align-items: center;
+  border-radius: 6px;
```

Comment:
What are you aligning the visual language with?

Reply:
using the border radius of 6px as same as githubs buttons